src/views/Home.js

```diff
@@ -33,6 +34,11 @@ class Home extends React.Component {
   render() {
     const {formatMessage} = this.props.intl;
     const {topHearing, openHearings, language} = this.props;
+    const hearingMap = (openHearings && openHearings.data ? (<div className="container map">
+      <h4><FormattedMessage id="open-hearings-on-map"/></h4>
+      <OverviewMap hearings={openHearings.data} style={{width: '100%', height: '40%'}} />
```

Comment:
Oh, and increase the height to 600px here as well, and the width to full screen width.